src/ast/value.rs

```diff
@@ -97,6 +97,32 @@ pub enum Value {
     Placeholder(String),
 }
 
+impl Into<String> for Value {
+    fn into(self) -> String {
+        match self {
+            Value::SingleQuotedString(s) => s,
+            Value::TripleSingleQuotedString(s) => s,
+            Value::TripleDoubleQuotedString(s) => s,
+            Value::EscapedStringLiteral(s) => s,
+            Value::UnicodeStringLiteral(s) => s,
+            Value::SingleQuotedByteStringLiteral(s) => s,
+            Value::DoubleQuotedByteStringLiteral(s) => s,
+            Value::TripleSingleQuotedByteStringLiteral(s) => s,
+            Value::TripleDoubleQuotedByteStringLiteral(s) => s,
+            Value::SingleQuotedRawStringLiteral(s) => s,
+            Value::DoubleQuotedRawStringLiteral(s) => s,
+            Value::TripleSingleQuotedRawStringLiteral(s) => s,
+            Value::TripleDoubleQuotedRawStringLiteral(s) => s,
+            Value::NationalStringLiteral(s) => s,
+            Value::HexStringLiteral(s) => s,
+            Value::DoubleQuotedString(s) => s,
+            Value::Placeholder(s) => s,
+            Value::DollarQuotedString(s) => s.value,
+            _ => panic!("not a string value"),
```

Comment:
Not sure about this one. This panics for Number, Boolean, Null. Should be safe if users only access String values where they are supposed to be?

Reply:
Strictly speaking no, but without it, users that used the string value before have to put code like this in their code. Thought it would be nice offering a more easy migration by just adding `.into()` to the existing usage.

Reply:
Thank you, that's a great suggestion. Have a look at my latest commit, does this make sense?